README.md

```diff
@@ -50,3 +50,26 @@ visualize('MyComponent', () => {
     xsnap('ignore this test', <MyComponent/>);
 });
 ```
+
+## Testing asynchronious actions
+
+In order for the asynchronious tests to work, `eyes-storybook` must be configured in a way it knows to wait for the component to notify it is done. This is done by setting a flag on the relevant story and inform applitools to look for this flag and wait.
+
+`storybook-snapper` provides a preconfigured `applitools.config.js` which you should use in your own `applitools.config.js` file.
+
+Change your `applitools.config.js` file to the following:
+```js
+const appName = require('./package').name;
```

Comment:
is this actually related to async tests?
shouldn't this be part of the default configs?

Reply:
I didn't want to start searching for the user's `package.json` file from within this library (like Yoshi does, for instance), so I thought it was better to get it from the users.
But we can use [`find-pkg`](https://www.npmjs.com/package/find-pkg), which is what Yoshi does.

Reply:
is this mandatory for the tool to work? if so, it should be built-in. if not - then why do we bother doing it?